calendar/page.js

```diff
@@ -82,31 +88,38 @@ class CalendarHandler {
       this.calendar.updateEvent(record.id, CALENDAR_NAME, {backgroundColor: CalendarHandler._selectedColor});
       this._selectedRecordId = record.id;
       this.calendar.setDate(record.startDate);
-      const dom = document.querySelector('.toastui-calendar-time');
-      const middleHour = record.startDate.getHours()
-        + (record.endDate.getHours() - record.startDate.getHours()) / 2;
-      dom.scrollTo({top: (dom.clientHeight / 24) * middleHour, behavior: 'smooth'});
+      if (this.calendar.getViewName() !== 'month') {
+        //Scoll to the middle of the event, if it's not month view
+        const dom = document.querySelector('.toastui-calendar-time');
+        const middleHour = record.startDate.getHours()
+          + (record.endDate.getHours() - record.startDate.getHours()) / 2;
+        dom.scrollTo({top: (dom.clientHeight / 24) * middleHour, behavior: 'smooth'});
+      }
     }
   }
 
   // change calendar perspective between week, month and day.
   changeView(calendarViewPerspective) {
     this.calendar.changeView(calendarViewPerspective);
+    document.getElementById('calendar-title').innerText = getMonthName();
   }
 
   // navigate to the previous time period
   calendarPrevious() {
     this.calendar.prev();
+    setMonthNameInDom();
```

Comment:
Perhaps a more general `updateAfterNavigation` name would work, could imagine needing to adapt the UI in several ways.